src/bloom.h

```diff
@@ -94,7 +94,26 @@ class CBloomFilter
  * insert()'ed ... but may also return true for items that were not inserted.
  *
  * It needs around 1.8 bytes per element per factor 0.1 of false positive rate.
- * (More accurately: 3/(log(256)*log(2)) * log(1/fpRate) * nElements bytes)
+ * e.g. for 1000 elements, we'd need:
+ *     ~1800 bytes for a false positive rate of 0.1
+ *     ~3600 bytes for a false positive rate of 0.01
+ *     ~5400 bytes for a false positive rate of 0.001
+ *
+ * This Python code calculates the exact filter size in bytes:
+
+import math
+def filterbytes(nel, fprate):
+    logfp = math.log(fprate)
+    nhash = max(1, min(round(logfp/math.log(0.5)), 50))
+    nfilbits = math.ceil(-1.0*nhash*(3*int((nel+1)/2)) / math.log(1.0 - math.exp(logfp / nhash)))
+    return (nfilbits+63)/64*2*8
+
+ * If we make these assumptions:
```

Comment:
Nit: I found these next 6 lines added complexity to something that was very clear thus far. Do you think the simpler formula makes it easier to reason about the size of the bloom filter?

Reply:
How does this render on doxygen? If it is markdown style you'd likely want to quote an inline script using triple-backquotes.

TBH I'm not sure the python script adds much in the first place. Sometimes a simple but good-enough approximation is better for documentation purposes. Is this level of precision necessary?

Reply:
Yeah, fair question @laanwj . I picked this up from the "up for grabs" #19130 but I could go either way—if we think the extra precision doesn't really help then we can close this PR.

Reply:
See my comment: https://github.com/bitcoin/bitcoin/pull/19130#issuecomment-637176259

Given how accurate the formula is in all reasonable cases (at most 0.115% off, and usually much less), I don't think it's worth including the code.

Reply:
Thanks for the input! Agreed that the extra complexity from the Python script is not worthwhile for the small accuracy gain; I'll update.

Reply:
Ooh, didn't know about doxygen use before, thanks for the reminder! The formatting should be reasonable now:

<img width="807" alt="Doxygen" src="https://user-images.githubusercontent.com/4276679/94890948-9a186580-0435-11eb-808e-4902ae092b69.png">